yoticlient.go

```diff
@@ -99,23 +94,25 @@ func (client *Client) GetSdkID() string {
 
 // GetActivityDetails requests information about a Yoti user using the one time use token generated by the Yoti login process.
 // It returns the outcome of the request. If the request was successful it will include the users details, otherwise
```

Comment:
I know you didn't change this, but I just noticed it
```suggestion
 // It returns the outcome of the request. If the request was successful it will include the user's details, otherwise
```